contracts/savings-app/src/msg.rs

```diff
@@ -24,17 +24,16 @@ pub struct AppInstantiateMsg {
 #[cfg_attr(feature = "interface", impl_into(ExecuteMsg))]
 pub enum AppExecuteMsg {
     /// Create the initial liquidity position
-    #[cfg_attr(feature = "interface", payable)]
     CreatePosition {
         lower_tick: i64,
         upper_tick: i64,
         // Funds to use to deposit on the account
         funds: Vec<Coin>,
         /// The two next fields indicate the token0/token1 ratio we want to deposit inside the current ticks
+        /// The ordering of these doesn't matter.
```

Comment:
It does matter because coin denoms have to be sorted alphabetically

Reply:
Contract logic will not figure out right now, see `tokens_to_swap` function. In current implementation it will return coins as `[asset0, asset1]`, without checking if they are sorted

Reply:
I see but that can be fixed by the TODO in the code, not an issue with the input of the assets.

Reply:
> I see but that can be fixed by the TODO in the code, not an issue with the input of the assets.

Added that todo without a fix because I'm not sure if we want to swap them or error in case they are in a wrong order. Pretty sure user supposed to provide asset0 and asset1 as stored in the CL pool, and not in reverse order.